pkgs/top-level/python-packages.nix

```diff
@@ -30460,28 +30463,46 @@ in {
     };
   };
 
-  # tensorflow is built from a downloaded wheel, because the upstream
-  # project's build system is an arcane beast based on
-  # bazel. Untangling it and building the wheel from source is an open
-  # problem.
+  # TensorFlow is built from a downloaded wheel, because the upstream
+  # project's build system is an arcane beast based on bazel. Untangling
+  # it and building the wheel from source is an open problem.
 
-  tensorflow = self.tensorflowNoGpuSupport;
+  tensorflow = self.tensorflowWithoutGpu;
 
-  tensorflowNoGpuSupport = buildPythonPackage rec {
+  tensorflowWithoutGpu = buildPythonPackage rec {
```

Comment:
As of now, TensorFlow does not support OpenCL. However, if we were to introduce `theanoWithGpu` and `theanoWithoutGpu`, I would suggest to use a similar naming here (for sake of consistency). What do you think?